lms/static/coffee/fixtures/autoenrollment.html

```diff
@@ -10,11 +10,11 @@
       <input disabled="disabled" id="browseFile" placeholder="choose file"/>
 
       <div class="file-browse btn btn-primary">
-        <span class="browse"> Browse </span>
-        <input class="file_field" id="browseBtn" name="students_list" type="file" accept=".csv"/>
+        <label for="browseBtn-auto_enroll_csv" class="browse">${_("Browse")}</label>
```

Comment:
Don't translate strings in fixture files. Somehow must comment on this must have gotten lost, or was it in a different PR?